requirements.txt

```diff
@@ -4,4 +4,6 @@ azure-identity
 quart
 quart-cors
 python-dotenv
-hypercorn
\ No newline at end of file
+hypercorn
+azure-ai-ml 
```

Comment:
I'm wondering if we shouldn't just move all of this to the nachet-model repo instead instead of introducing unrelated dependencies to our backend server?

Reply:
Done